### DISCUSSION TITLE
Finding eigenvectors

### DISCUSSION THREAD

**Discussion by hyeonjin-cho** [satijalab/seurat#4506] [Q&A] [answered]
Hi, I'm using Seurat to perform PCA analysis for my single-cell data, and I want to find where eigenvectors are stored, for the purpose of my research project. Is there anywhere where Seurat stores its eigenvectors? 

Thank you!

**Comment by timoast** [comment, marked as answer]
These can be accessed using the `Embeddings()` and `Loadings()` functions
